src/Jaeger/Configuration.cs

```diff
@@ -272,6 +284,12 @@ public Configuration WithTracerTags(Dictionary<string, string> tracerTags)
             return this;
         }
 
+        public Configuration WithExpandExceptionLogs(bool expandExceptionLogs=true)
```

Comment:
Not sure if we should do it using the default value. The idea was to keep it similar to [Tracer.Builder.WithExpandExceptionLogs()](https://github.com/jaegertracing/jaeger-client-csharp/blob/68dac3108ed4c020245a58ce484ba9722ec6981e/src/Jaeger/Tracer.cs#L295), but also have it resettable.